libcxx/test/libcxx/ranges/range.adaptors/range.stride.view/iterator/increment.pass.cpp

```diff
@@ -0,0 +1,27 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+// UNSUPPORTED: no-exceptions
+// UNSUPPORTED: !libcpp-hardening-mode=debug
+// XFAIL: availability-verbose_abort-missing
+
+// constexpr stride_view::<iterator>& operator++() {
+
+#include <ranges>
+
+#include "check_assertion.h"
+
+int main(int, char**) {
+  int range[]   = {1, 2, 3};
```

Comment:
We need to test these operations both inside `constexpr` and outside `constexpr`. We have a pattern where we define a `constexpr bool test() {}` function you can find in many places.

This comment applies everywhere.

Reply:
Understood! Thanks! I will address ASAP!

Reply:
As I look at other tests in the `libcxx`-specific section, I am not 100% sure that we can follow this pattern in this situation. The other places where `TEST_LIBCPP_ASSERT_FAILURE` are used, they are only used outside a `constexpr` context. Is there something that I am missing?